packages/eslint-plugin-calcite-components/tests/lib/rules/enforce-ref-last-prop/enforce-ref-last-prop.good.tsx

```diff
@@ -0,0 +1,22 @@
+// @ts-nocheck
+@Component({ tag: "sample-tag" })
+export class SampleTag {
+  render() {
+    return (
+      <Host>
+        <div
+          class="some-class"
```

Comment:
Suggestion: could you also add `tabIndex` to tie in the [scenario](https://github.com/jcfranco/stencil-ref-called-in-prop-order-instead-of-after-node-being-initialized/blob/main/src/components/my-component/my-component.tsx#L12-L15) that this rule helps avoid?

Reply:
That's a good point! ☉